src/Features/Core/Portable/Shared/TestHooks/Legacy/ListenerForwarders.cs

```diff
@@ -1,16 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-#nullable disable
-
-using System.Runtime.CompilerServices;
-
-[assembly: TypeForwardedTo(typeof(Microsoft.CodeAnalysis.Shared.TestHooks.AsynchronousOperationListener))]
-[assembly: TypeForwardedTo(typeof(Microsoft.CodeAnalysis.Shared.TestHooks.AsynchronousOperationListenerProvider))]
```

Comment:
We used to have things consuming this via IVT? We're sure they're not around anymore?

Reply:
IDK. How do I find out?

Reply:
Kicked off val build to see if anything fails:
https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=11030936&view=results

Reply:
Working on removing usage from TypeScript